package.json

```diff
@@ -125,7 +125,7 @@
     },
     "dependencies": {
         "@0x/assert": "^3.0.17",
-        "@0x/asset-swapper": "^5.2.0",
+        "@0x/asset-swapper": "0xProject/gitpkg-registry#0x-asset-swapper-v5.2.0-0c08353b2",
```

Comment:
Can you link to the asset-swapper changes? was it just the naming change?

Reply:
Yes indeed! https://github.com/0xProject/protocol/pull/63